charts/k8ssandra-cluster/templates/cleaner/batch_job.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Name }}-cleaner-job-k8ssandra
+  labels:
+{{ include "k8ssandra-cluster.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
+spec:
+  backoffLimit: 3
+  template:
+    metadata:
+      labels:
+{{ include "k8ssandra-cluster.labels" . | indent 8 }}
+    spec:
+      restartPolicy: Never
+      serviceAccountName: {{ .Release.Name }}-cleaner-k8ssandra
+      containers:
+        - name: cleaner-job-k8ssandra
+          image: {{ .Values.cleaner.image }}
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+          args:
+            - k8ssandra-client
```

Comment:
What is the purpose of this first argument, `k8ssandra-client`? I don't think I see it actually used.

Reply:
Removing will make the first parameter fail as that job runner will remove the first parameter.